src/templates/pad.html

```diff
@@ -64,12 +64,12 @@
 
             <ul class="menu_left" role="toolbar">
                 <% e.begin_block("editbarMenuLeft"); %>
-                <%- toolbar.menu(settings.toolbar.left, isReadOnly) %>
+                <%- toolbar.menu(settings.toolbar.left, isReadOnly, 'left') %>
```

Comment:
In `pad.html` it gets the values `'left'` or `'right'` (two string literals).
Fine.